File: loianeTraining/javaBasico/exercicios-aulas-16-17/Ex12.java
import java.util.Scanner;

public class Ex12 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        System.out.println("Tabuada de qual numero deseja ver?");
        var numeroDaTabuada = scan.nextInt();
        for (int i = 0; i <= 10; i++) {
            System.out.println(numeroDaTabuada + " x " + i + " = " + numeroDaTabuada * i);
        }
        scan.close();
    }
}

File: Loiane_Training_C#/Basico/D/Ex12.cs
using System;

class Ex12
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite qual tabuada, entre 1 e 10, deseja ver: ");
        int numeroDaTabuada = int.Parse(Console.ReadLine());

        if (numeroDaTabuada > 0 && numeroDaTabuada < 10)
        {
            for (int i = 1; i <= 10; i++)
            {
                Console.WriteLine($"{numeroDaTabuada} x {i} = {numeroDaTabuada * i}");
            }
        }
        else
        {
            Console.WriteLine("Entrada Invalida!");
        }
    }
}